docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -83,107 +83,113 @@ configuration file as documented under
-
+- `<public-key>` is the public key of the node to be added/updated/removed. The encoding and type of the key have to match those specified in `genesis.json` in the supported Tendermint version.
+- `<power>` is the new power for the validator. To remove the validator, set the power to `0`.
+- `<node-id>` is the node identifier from Tendermint. A node operator can learn his node identifier by executing `tendermint show_node_id`.
+- `<path-to-the-private-key>` is the path to the private key of the validator who proposes the election. Tendemint places it at  `.tendermint/config/priv_validator.json`.
 
-NOTE: A change to the validator set can only be proposed by one of the exisitng validators.
-
-Example usage,
+Example:
 
 ```bash
 $ bigchaindb election new upsert-validator HHG0IQRybpT6nJMIWWFWhMczCLHt6xcm7eP52GnGuPY= 1 fb7140f03a4ffad899fabbbf655b97e0321add66 --private-key /home/user/.tendermint/config/priv_validator.json
 [SUCCESS] Submitted proposal with id: 04a067582cf03eba2b53b82e4adb5ece424474cbd4f7183780855a93ac5e3caa
 ```
 
-A successful execution of the above command **doesn't** imply that the validator set will be immediately updated but rather it means the proposal has been succcessfully accepted by the network. Once the `election_id` has been generated the node operator should share this `election_id` with other validators in the network and urge them to approve the proposal. Note that the node operator should themsleves also approve the proposal.
+A successful execution of the above command does not imply the validator set has been updated but rather the proposal has been accepted by the network.
+Once `election_id` has been generated, the proposer should share it with other validators of the network (e.g. via email) and ask them to approve the proposal.
+
+Note that election proposers do not automatically approve elections by proposing them.
 
+For more details about how validator set changes work, refer to [BEP-21](https://github.com/bigchaindb/BEPs/tree/master/21).
 
-###### election new migration
+###### election new chain-migration
 
-Create an election to halt block production, to allow for a version change across breaking changes.
+Create an election to halt block production, to coordinate on making a Tendermint upgrade with a backwards-incompatible chain.
 
 
 ```bash
-$ bigchaindb election new migration --private-key PATH_TO_YOUR_PRIVATE_KEY
-[SUCCESS] Submitted proposal with id: <election_id>
+$ bigchaindb election new chain-migration --private-key <path-to-theprivate-key>
 ```
 
-- `--private-key`: The path to Tendermint's private key which can be generally found at `/home/user/.tendermint/config/priv_validator.json`.
+- `<path-to-the-private-key>` is the path to the private key of the validator who proposes the election. Tendemint places it at  `.tendermint/config/priv_validator.json`.
```

Comment:
Tendemint --> Tendermint, both here and elsewhere. (Do a search for "Tendemint".)